src/protocols/at-ps2/keyboard_interface.c

```
@@ -74,11 +75,19 @@
  *   normal Ctrl+key combinations is not safe. The recommended workaround is to
  *   remap an unused physical key (for example Scroll Lock) to PAUS in the
  *   keyboard configuration for that physical keyboard.
+ *
+ * @see keyboard_interface.h for the public API and keyboard command protocol constants.
  */
 
 #include "keyboard_interface.h"
 
+#include <stdbool.h>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)